tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -135,50 +135,58 @@ is usually:
+| inout mat4 **MODELVIEW_MATRIX**      | Model space to view space transform (use if possible). |
++--------------------------------------+--------------------------------------------------------+
+| inout mat4 **INV_PROJECTION_MATRIX** | Clip space to view space transform.                    |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_WORLD**      | Node position, in world space.                         |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **NODE_POSITION_VIEW**       | Node position, in view space.                          |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_POSITION_WORLD**    | Camera position, in world space.                       |
++--------------------------------------+--------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**   | Camera direction, in world space.                      |
++--------------------------------------+--------------------------------------------------------+
+| inout vec3 **VERTEX**                | Vertex in local coordinates.                           |
++--------------------------------------+--------------------------------------------------------+
+| out vec4 **POSITION**                | If written to, overrides final vertex position.        |
++--------------------------------------+--------------------------------------------------------+
+| inout vec3 **NORMAL**                | Normal in local coordinates.                           |
++--------------------------------------+--------------------------------------------------------+
+| inout vec3 **TANGENT**               | Tangent in local coordinates.                          |
++--------------------------------------+--------------------------------------------------------+
+| inout vec3 **BINORMAL**              | Binormal in local coordinates.                         |
++--------------------------------------+--------------------------------------------------------+
+| out float **ROUGHNESS**              | Roughness for vertex lighting.                         |
++--------------------------------------+--------------------------------------------------------+
+| inout vec2 **UV**                    | UV main channel.                                       |
++--------------------------------------+--------------------------------------------------------+
+| inout vec2 **UV2**                   | UV secondary channel.                                  |
++--------------------------------------+--------------------------------------------------------+
+|| in bool **OUTPUT_IS_SRGB**          || ``true`` when calculations happen in sRGB color space |
+||                                     || (``true`` in GLES2, ``false`` in GLES3).              |
```

Comment:
Double `|`s

Reply:
I can always count on you :D thanks for noticing